projects/ADuCM3029_demo_ad5592r_ad5593r/src/aio_dio_pdmz.c

```diff
@@ -100,6 +105,46 @@ static const uint16_t staircase[] = {812, 1638, 2458, 3277};
 /************************ Functions Definitions *******************************/
 /******************************************************************************/
 
+/**
+ * Convert floating point value to ASCII. Maximum 8 decimals.
+ *
+ * @param [in] value   - The floating point value to be converted.
+ * @param [out] buffer - ASCII buffer; output buffer.
+ *
+ * @return 0 in case of success, negative error code otherwise.
+ */
+static void aiodio_ftoa(uint8_t *buffer, float value)
```

Comment:
you can use sprintf((char *)buffer, "%.8f", value) instead of writing your own function